homeassistant/components/lutron/config_flow.py

```diff
@@ -0,0 +1,65 @@
+"""Config flow to configure the Lutron integration."""
+from __future__ import annotations
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import DOMAIN
+
+
+class LutronConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """User prompt for Main Repeater configuration information."""
+
+    VERSION = 1
+
+    async def async_step_user(self, user_input=None) -> FlowResult:
+        """First step in the config flow."""
+        errors = {}
+
+        if user_input is not None:
+            # Validate user input
+            username = user_input.get(CONF_USERNAME)
+            password = user_input.get(CONF_PASSWORD)
+            ip_address = user_input.get(CONF_HOST)
+
+            # Perform any validation here
+            if not username or not password or not ip_address:
+                errors["base"] = "missing_fields"
+            else:
```

Comment:
you can expect the submitted data to follow your Schema, so since you put them as required you can expect them to be filled.

Reply:
removed redundant validation